SETUP/generate_css_from_less.sh

```diff
@@ -40,11 +37,11 @@ function run_less {
     css_file="${less_file%.*}.css"
     if [ $CHECK -eq 0 ]; then
         echo "Generating $css_file"
-        lessc $less_file $css_file
+        ../node_modules/.bin/lessc $less_file $css_file
```

Comment:
despite the path, my system was still picking the system on instead of the node one.